app/scripts/modules/core/pipeline/config/triggers/docker/dockerTrigger.html

```diff
@@ -3,11 +3,11 @@
     <div class="col-md-3 sm-label-right">
       Registry
     </div>
-    <div class="col-md-5">
-      <ui-select ng-model="trigger.registry" class="form-control input-sm">
```

Comment:
replaced registry with account, will make corresponding changes to igor and echo downstream